chapter3/tests/t11/camp-php.local

```diff
@@ -0,0 +1,25 @@
+server {
```

Comment:
in future pls prefer single nginx.conf file - much easier to review.

Reply:
Got it!